docs/book/localization.md

```diff
@@ -339,18 +339,16 @@ previous method under the [Project Structure](#project-structure) section.
 
 :::tip Note
 This method only localizes utterances and say/reprompt strings. Strings in code
-are not captured; this means that slotbuilder functions, `context.say.push` calls,
-function calls that may use strings, and inline slot definitions are not
-localized. You will need to handle localization in these functions themselves.
-You can pass `context.language` to code to retrieve the Litexa language.
+are not captured; if you use slot builders, inline slot definitions, or inline
```

Comment:
Slot builders and inline slot definitions would basically be the same thing, no? If so, would consolidate.